api.py

```diff
@@ -0,0 +1,62 @@
+from typing import Optional
+import sublime
+from pieces import __version__
+import pieces_os_client as pos_client
+from pieces import config
+import time
+application = None
+
+def get_version() -> Optional[str]:
+    """Get pieces os version return None if there is a problem"""
+    try:
+        version = pos_client.WellKnownApi(config.api_client).get_well_known_version()
+        return version
+    except: # There is a problem in the startup
+        return None
+
+
+def get_health():
+    try:
+        health = pos_client.WellKnownApi(config.api_client).get_well_known_version()
+        return health
+    except: # There is a problem in the startup
+        return False
+
+
+def open_pieces_os() -> Optional[str]:
+    """Open pieces os and return its version"""
+    version = get_version()
+    if version:
+        return version
+    else:
+    	# sublime.platform() ->  Literal['osx', 'linux', 'windows']
+        pl = sublime.platform().upper()
+        if pl == "windows":
+            subprocess.Popen(["start", "os_server"], shell=True)
+        elif pl == "linux":
+            subprocess.Popen(["os_server"])
+        elif pl == "osx":
+            subprocess.Popen(["open", "os_server"])
+        time.sleep(2) # wait for the server to open
+        
+        return get_version() # pieces os version
+
+def get_application() -> pos_client.Application:
```

Comment:
most likely want this to exist as a class so we don't rely on globals

Reply:
Alr